projects/Unstructured Supplemenrary  Service Data/ussdtim.py

```diff
@@ -0,0 +1,194 @@
+import random
+import time
+import sys
+
+print('Welcome To fastrack USSD Banking Project...')
+time.sleep(8)
+
+bank_list="""
+1. Access Bank
+2. Fidelity Bank
+3. Guarantee Trust Bank
+4. Heritage Bank
+5. Polaris Bank
+6. Stanbic IBTC
+7. Unity Bank
+8. Wema Bank
+"""
+
+gen_bvn = " "
+
+def BVN_checker( ):
+    global gen_bvn
+    bvn = [str(i) for i in range (5)]
+    gen_bvn= "".join(bvn)
+    print (gen_bvn)
+       				
+def open_acct( ):
+	global gen_bvn
+	print("Welcome to our online Account opening services.")
+	print("loading...")
+# creating an empty list to serve as a temporary place holder.
+	temp_storage= [ ]
+	f_name= input("Enter your first name:")
+	s_name= input ("Enter your second name:")
+	sex = input("Enter sex [M/F]:")
+	BVN_checker( )
+	temp_storage.extend(f_name)

```

Comment:
it will convert your input to list and append to list
Example:
```python
temp_storage = []
f_name = "Roy"
temp_storage.extend(f_name)

# print temp_storage
print(temp_storage)

'''
["R","o","y"]
'''
```
correct it..

Reply:
Ok, will do this

Reply:
@chavarera , append works better because of the spacing of words